app/src/main/java/com/example/iou/notifications/NotificationUtils.java

```diff
@@ -58,9 +59,12 @@ public NotificationCompat.Builder setNotification(String title, String body) {
     }
 
     private void initializeIntent() {
+
         mapIntent = new Intent(this, MainActivity.class);
         mapIntent.putExtra(FRAGMENT_KEY, "map");
```

Comment:
You may want to be put these fragment names into a shared location as well, since they're referenced from multiple places. You could even create an enum value for them if you'd like!